code401-challenges/src/test/java/code401/challenges/tree/BinarySearchTreeTest.java

```diff
@@ -0,0 +1,42 @@
+package code401.challenges.tree;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class BinarySearchTreeTest {
+    BinarySearchTree classUnderTest;
+    Node<Integer> root;
+    @Before
+    public void testInstantiateClass(){
+        classUnderTest = new BinarySearchTree();
+    }
+
+    @Test
+    public void testInstantiateEmptyTree(){
+        assertNull("This should return true if the tree is empty/null ", classUnderTest.getRoot());
+    }
+
+    @Test
+    public void testSingleRootNodeTree(){
```

Comment:
This root doesn't actually test if the tree contains one node, it checks to see if the tree contains a node that has a value of 1. To check if this tree has one node you could have checked that the `root` wasn't `null` and that that the `root`s left and right were also `null`